Documentation/interact_v3.md

```diff
@@ -0,0 +1,182 @@
+# Interact with etcd
+
+Users mostly interact with etcd by putting or getting the value of a key. This section describes how to do that by using etcdctl, a command line tool for interacting with etcd server. The concepts described here should apply to the gRPC APIs or client library APIs.
+
+By default, etcdctl talks to etcd server with v2 API for backward compatibility reason. To make etcdctl talks to etcd with v3 API, we first need to set API version to 3 by setting environment. 
```

Comment:
By default, etcdctl talks to the etcd server with the v2 API for backward compatibility. For etcdctl to speak to etcd using the v3 API, the API version must be set to version 3 via the `ETCDCTL_API` environment variable.